src/main/java/net/dries007/tfc/objects/blocks/agriculture/BlockCropDead.java

```diff
@@ -0,0 +1,127 @@
+package net.dries007.tfc.objects.blocks.agriculture;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+import javax.annotation.Nonnull;
+
+import net.minecraft.block.BlockBush;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.properties.PropertyBool;
+import net.minecraft.block.state.BlockStateContainer;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+import net.minecraftforge.common.EnumPlantType;
+
+import net.dries007.tfc.api.types.ICrop;
+import net.dries007.tfc.objects.items.ItemSeedsTFC;
+
+public class BlockCropDead extends BlockBush
+{
+    /* true if the crop spawned in the wild, means it ignores growth conditions i.e. farmland */
+    public static final PropertyBool WILD = PropertyBool.create("wild");
+
+    // binary flags for state and metadata conversion
+    private static final int META_WILD = 1;
+
+    // static field and methods for conversion from crop to Block
+
+    private static final Map<ICrop, BlockCropDead> MAP = new HashMap<>();
+
+    public static BlockCropDead get(ICrop crop)
+    {
+        return MAP.get(crop);
+    }
+
+    public static Set<ICrop> getCrops()
+    {
+        return MAP.keySet();
+    }
+
+    // fields
```

Comment:
These are unnecessary comments. Our code style already sorts various class members so there's no need to delimit sections, and the members themselves tell you what they are. i.e. a constructor is obviously not a field.

Reply:
I've removed all the comments I've found and reordered functions according to the reformat code style.